scss/_variables.scss

```diff
@@ -1614,16 +1614,20 @@ $pagination-transition:             $transition-focus !default;
 // Boosted mod: no $pagination-border-radius-lg
 
 // Boosted mod
-$pagination-padding-end:            1.125rem !default;
 $pagination-icon:                   var(--#{$prefix}chevron-icon) !default;
-$pagination-icon-size:              subtract($spacer * 2, calc(var(--#{$prefix}border-width) * 2)) !default; // stylelint-disable-line function-disallowed-list
+$pagination-icon-min-width:         .625rem !default;
+$pagination-icon-size:              1.125rem !default;
```

Comment:
These two should be named almost the same since their goal is the same. Maybe something like `$pagination-icon-efficient-width/height`? It add some changes to variables but it feel more handy to me. What would think about a change like this ?

Reply:
Something like `$pagination-icon-hitbox-width/height`? Or `$pagination-icon-box-width/height` maybe?
Not sure which one sounds more like a Bootstrap's name?

Reply:
I don't have any idea on how to call those. Let's wait for the last review to decide ?

Reply:
Yes, it will be a job for the led dev review 😉 

**Simple reminder:** changing the name of `$pagination-icon-size` will need to be properly documented in our migration guide since it's an already existing variable.